teams/infra/pull-request/README.md

```diff
@@ -3,6 +3,31 @@ This collection of github actions is currently intended to enable `Auto-Releases
 
 If applied correctly, every commit to the mainline branch of a repository will be [added to a Release PR](./release/README.md) which then can be [automerged](./automerge/README.md) on a schedule.
 
+## Available Actions
+
+This package contains three complementary actions for pull request management:
+
+* **[Release](./release/README.md)** - Automated release PR creation and management based on conventional commits
+* **[Automerge](./automerge/README.md)** - Automated merging of release PRs on a schedule
+* **[Validate Title](./validate-title/README.md)** - Validates PR titles against conventional commit format (recommended for side projects)
```

Comment:
change the recommendation to: Projects which use release-please to manage releases.